onnxruntime/core/framework/session_state.h

```diff
@@ -145,7 +145,7 @@ class SessionState {
      * This function will take a shallow copy of d if d is not NULL.
      * If 'constant' is true the tensor value cannot be overridden by an input at runtime.
      */
```

Comment:
Comment needs updating, especially around why we need an argument for `sparse` when there's OrtValue::IsSparseTensor.

Reply:
Will update. Currently all initializers are stored in Tensors, so OrtValue::IsSparseTensor will return false. This will change when we store them as SparseTensors